app/src/main/java/org/sopt/official/feature/auth/component/AuthTextWithArrow.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * MIT License
+ * Copyright 2025 SOPT - Shout Our Passion Together
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.sopt.official.feature.auth.component
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import org.sopt.official.R
+import org.sopt.official.designsystem.Gray30
+import org.sopt.official.designsystem.SoptTheme
+
+@Composable
+internal fun AuthTextWithArrow(
+    text: String,
+    modifier: Modifier = Modifier,
+    textStyle: TextStyle = SoptTheme.typography.label14SB,
+    textColor: Color = Gray30,
+) {
```

Comment:
```suggestion
@Composable
internal fun AuthIndicatorText(
    text: String,
    modifier: Modifier = Modifier,
    textStyle: TextStyle = SoptTheme.typography.label14SB,
    textColor: Color = Gray30,
) {
```
정도는 어떨까요? @Hyobeen-Park @angryPodo 좋은 네이밍 추천받습니다.

Reply:
`AuthNavigationText`는 어떤지...?

Reply:
오 `AuthNavigationText` 좋은데요?

Reply:
다이얼로그로 (혹은 바로) 다른 화면의 이동을 도와주는 텍스트라 `AuthNavigationText` 좋은 것 같습니다. 감사해요!